view/src/components/produce.js

```diff
@@ -0,0 +1,723 @@
+                                  Maintenance Temperature:{" "}
+                                  {produce.shippingMaintenanceTemperatureLow} -{" "}
+                                  {produce.shippingMaintenanceTemperatureHigh}
+                                  <br />
+                                  Preset Temperature:{" "}
+                                  {produce.shippingPresetTemperature}
+                                </Typography>
+                              </Box>
+                              <Box p={3}>
+                                <Typography
+                                  className={classes.pos}
+                                  color="textSecondary"
+                                >
+                                  Pricing (in USD / lb):
+                                </Typography>
+                                <Typography variant="body2" component="p">
+                                  USDA Price: ${produce.price}
+                                  <br />
+                                  Average Price Paid by Farmlink: $
+                                  {produce.pricePaid}
+                                </Typography>
+                              </Box>
+                              <Box p={3}>
+                                <Typography
+                                  className={classes.pos}
+                                  color="textSecondary"
+                                >
+                                  Internal Statistics:
+                                </Typography>
+                                <Typography variant="body2" component="p">
+                                  Amount Moved by FarmLink (lbs):{" "}
+                                  {produce.amountMoved}
+                                </Typography>
+                              </Box>
+                            </Box>
+                          </CardContent>
+                          <CardActions>
+                            <Button
+                              size="small"
+                              color="primary"
+                              onClick={() => this.handleViewOpen({ produce })}
+                            >
+                              {" "}
+                              View{" "}
+                            </Button>
+                            <Button
+                              size="small"
+                              color="primary"
+                              onClick={() =>
+                                this.handleEditClickOpen({ produce })
+                              }
+                            >
+                              Edit
+                            </Button>
+                            <Button
+                              size="small"
+                              color="primary"
+                              onClick={() =>
+                                this.deleteTodoHandler({ produce })
```

Comment:
I think that `deleteTodoHandler` should be `deleteProduceHandler`.

Reply:
(@andrewhojel )